tests/test-wp-parsely.php

```diff
@@ -250,7 +250,19 @@ function test_http_canonicals()
         $ppage = self::$parsely->insert_parsely_page();
         $this->assertTrue(strpos($ppage['url'], 'https', 0) == 0);
     }
-
+    
+    
+      
+    function test_amp_analytics() {  
+      	$options = get_option('parsely');
+        $post_array = $this->create_test_post_array();
+        $post = $this->factory->post->create($post_array);
+        $this->go_to('/?p=' . $post);
+        $ppage = self::$parsely->insert_parsely_amp_tracking();
```

Comment:
Instead of generating the `ppage` object, let's change this line to `$output = self::$parsely->parsely_add_amp_actions();`, so that we can test just for the existence of the parsely amp analytics code